examples/intel_extension_for_pytorch/README.md

```diff
@@ -90,6 +92,36 @@ Add the following lines in `config.properties` to use launcher with its default
 cpu_launcher_enable=true
 ```
 
+## TorchServe with Intel® Extension for PyTorch* and Intel GPUs
```

Comment:
Can you please provide the information link about intel GPU performance gain?

Reply:
Added the intel GPU performance gain to README.md.